interface/main/messages/templates/linked_documents.php

```diff
@@ -0,0 +1,237 @@
+    return;
+}
+
+$enc_list = [];
+if (!empty($prow)) {
+    $results = QueryUtils::fetchRecords("SELECT fe.encounter,fe.date,openemr_postcalendar_categories.pc_catname FROM form_encounter AS fe " .
+        " LEFT JOIN openemr_postcalendar_categories ON fe.pc_catid=openemr_postcalendar_categories.pc_catid  WHERE fe.pid = ? ORDER BY fe.date DESC", array($prow['pid']));
+    foreach ($enc_list as $row) {
+        $enc_list[] = [
+            'encounter' => $row['encounter'],
+            'pc_catname' => xl_appt_category($row['pc_catname']),
+            'date' => oeFormatShortDate(date("Y-m-d", strtotime($row['date'])))
+        ];
+    }
+}
+
+// if we have phimail enabled, we are going to find out if our document is an xml file, if so, we are going to do a validation
+// report on the document.  This would be a great thing for caching if we wanted to do that, we'll just compute on the fly for now.
+$records = [];
+$prow = $prow ?? null;
+$cdaDocumentValidator = new CdaValidateDocumentObject();
+foreach ($tmp as $record) {
+    $d = new Document($record['id1']);
+    $docInformation = [
+        'documentId' => $d->get_id(),
+        'title' => $d->get_name() . "-" . $d->get_id(),
+        'isCda' => false,
+        'hasPatient' => false,
+        'requiresValidation' => false
+    ];
+    if (!empty($prow)) {
+        $docInformation['hasPatient'] = true;
+        $docInformation['pname'] = $prow['fname'] . " " . $prow['lname'];
+        $docInformation['pid'] = $prow['pid'];
+        $docInformation['pubpid'] = $prow['pubpid'];
+        $docInformation['DOB'] = $prow['DOB'];
+    }
+    if ($cdaDocumentValidator->isCdaDocument($d)) {
+        $docInformation['isCda'] = true;
+        $docInformation['requiresValidation'] = true;
+    }
+    $records[] = $docInformation;
+}
+$twig = new TwigContainer(null, $GLOBALS['kernel']);
+try {
+    foreach ($records as $record) : ?>
+<div class="row mt-2 mb-2 messages-document-row <?php echo $record['requiresValidation'] ? "messages-document-validate" : ""; ?>"
+     data-doc="<?php echo $record['documentId']; ?>">
+    <div class="col-12">
+    <span class='font-weight-bold'><?php echo xlt('Linked document'); ?>:</span>
+        <?php if (!$record['hasPatient']) : ?>
+        <a class='messages-document-link<?php echo $record['requiresValidation'] ? " d-none" : ""; ?>'
+           href='javascript:void(0);' onClick="previewDocument(<?php echo attr_js($record['documentId']); ?>);"
+
+        >
+            <?php echo text($record['title']); ?>
+        </a>
+        <?php else : ?>
+            <a class='messages-document-link <?php echo $record['requiresValidation'] ? "d-none" : ""; ?>'
+               href='javascript:void(0);'
+               onClick="gotoReport(<?php echo attr_js($record['documentId']); ?>, '<?php echo attr_js($record['pname']); ?>', '<?php echo attr_js($record['pid']); ?>','<?php echo attr_js($record['pubpid'] ?? $record['pid']); ?>','<?php echo attr_js($record['DOB']); ?>');">
```

Comment:
remove the single quotes from all the attr_js calls here (so far have only been removed from 1st parameter)

Reply:
fixed.